src/towncrier/newsfragments/507.misc

```diff
@@ -0,0 +1 @@
+Provide a friendly message if --version option is required (for non-Python projects that don't explicitly provide the version in the config)
```

Comment:
```suggestion
A friendly message is now provided, when it's necessary to pass the ``--version`` option explicitly.
```

Let's not over-explain, since it might grow stale / miss something.